### PR TITLE
fix(simulated/series): fill pools taking into account decimals

### PR DESCRIPTION
**Pull request by abylaw**
Closes GEN-772

### CODE REVIEW

**Review by jparklev — APPROVED** [review]
ty! looks like the build is failing with this, but otherwise lgtm

Screen Shot 2022-03-24 at 8 37 10 AM

File: pkg/deployments/deploy/simulated/06-series.js [hunk 1]
```diff
@@ -76,10 +73,14 @@ module.exports = async function () {
       const { abi: tokenAbi } = await deployments.getArtifact("Token");
       const pt = new ethers.Contract(ptAddress, tokenAbi, signer);
       const yt = new ethers.Contract(ytAddress, tokenAbi, signer);
+      const decimals = await target.decimals()
+      const one = ethers.BigNumber.from(1).mul(ethers.BigNumber.from(10).pow(decimals))
+      const fourtyThousand =  ethers.BigNumber.from(40_000).mul(ethers.BigNumber.from(10).pow(decimals))
+      const oneMillion = ethers.BigNumber.from(1_000_000).mul(ethers.BigNumber.from(10).pow(decimals))
```
Comment (fedealconada) [inline pkg/deployments/deploy/simulated/06-series.js:79]: fun to see the different ways we can implement this hehe --> on the Capital Seeder I've used parseUnits-> https://github.com/sense-finance/sense-v1-autotasks/blob/3b73c3744c83e4d798e5fcd7a96d3991c2571578/utils/data.ts#L134

**Review by fedealconada — APPROVED** [review]
LGTM! Thanks for the PR!

File: pkg/deployments/deploy/simulated/06-series.js (hunk 1 above)
Comment (abylaw) [reply pkg/deployments/deploy/simulated/06-series.js]: oh that looks quite clean! i changed it to your approach hehe

**Pull request merged**